generators/typescript-axios/templates/configuration.mustache

```diff
@@ -0,0 +1,112 @@
+/* tslint:disable */
+/* eslint-disable */
+{{>licenseInfo}}
+
+{{! Added code - BEGIN }}
+import { BASE_PATH } from "./base";
+
+export enum Region{
```

Comment:
I'm still puzzled why this is needed to be custom, given we defined the different api urls in the OpenAPI